src/ssl.c

```diff
@@ -62668,12 +62678,7 @@ PKCS7* wolfSSL_d2i_PKCS7_ex(PKCS7** p7, const unsigned char** in, int len,
     if ((pkcs7 = (WOLFSSL_PKCS7*)wolfSSL_PKCS7_new()) == NULL)
         return NULL;
 
-    if (GetSequence(*in, &idx, &pkcs7->len, len) < 0) {
-        wolfSSL_PKCS7_free((PKCS7*)pkcs7);
-        return NULL;
-    }
-    pkcs7->len += idx;
-
+    pkcs7->len = len;
```

Comment:
Not sure if necessary, but I wonder if we should check that `len` is > 0.

Reply:
Sure, I'll add this check.